### PR TITLE
feat: add performance measurement range value

### PR DESCRIPTION
**Pull request by jing-zhi**
#### What this PR does / why we need it:

feat: add performance measurement range value

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=516226&iterationID=12560&tab=TASK&type=TASK)

#### Specified Reviewers:

/assign @chengjoey @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      feat: add performance measurement range value        |
| 🇨🇳 中文    |       支持自然年，上下半年维度的效能度量查询       |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by chengjoey — APPROVED** [review]

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go [hunk 1]
```diff
@@ -29,59 +29,100 @@ import (
...
+	OrgName                         string  `json:"orgName" ch:"orgName"`
+	ProjectID                       string  `json:"projectID" ch:"projectID"`
+	UserPosition                    string  `json:"userPosition" ch:"userPosition"`
+	UserPositionLevel               string  `json:"userPositionLevel" ch:"userPositionLevel"`
+	JobStatus                       string  `json:"jobStatus" ch:"jobStatus"`
+	ProjectName                     string  `json:"projectName" ch:"projectName"`
+	ProjectDisplayName              string  `json:"projectDisplayName" ch:"projectDisplayName"`
+	RangeRequirementTotal           float64 `json:"rangeRequirementTotal" ch:"rangeRequirementTotal"`
+	RequirementTotal                float64 `json:"requirementTotal" ch:"requirementTotal"`
+	RangeWorkingRequirementTotal    float64 `json:"rangeWorkingRequirementTotal" ch:"rangeWorkingRequirementTotal"`
```
Comment (sfwn) [inline internal/apps/dop/providers/efficiency_measure/efficiency_query.go:46]: Too many changes, separate into groups.

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go [hunk 2]
```diff
@@ -180,34 +221,63 @@ func (p *provider) makeEfficiencyBasicSql(req *apistructs.PersonalEfficiencyRequ
...
 	       last_value(number_field_values[indexOf(number_field_keys,'personal_pending_requirement_total')]) as pendingRequirementTotal,
+	       first_value(number_field_values[indexOf(number_field_keys,'personal_task_total')]) as firstTaskTotal,
 	       last_value(number_field_values[indexOf(number_field_keys,'personal_task_total')]) as taskTotal,
+	       first_value(number_field_values[indexOf(number_field_keys,'personal_working_task_total')]) as firstWorkingTaskTotal,
 	       last_value(number_field_values[indexOf(number_field_keys,'personal_working_task_total')]) as workingTaskTotal,
+	       first_value(number_field_values[indexOf(number_field_keys,'personal_pending_task_total')]) as firstPendingTaskTotal,
 	       last_value(number_field_values[indexOf(number_field_keys,'personal_pending_task_total')]) as pendingTaskTotal,
+	       first_value(number_field_values[indexOf(number_field_keys,'personal_bug_total')]) as first_bug_total,
 	       last_value(number_field_values[indexOf(number_field_keys,'personal_bug_total')]) as bug_total,
+           first_value(number_field_values[indexOf(number_field_keys,'personal_owner_bug_total')]) as first_owner_bug_total,
```
Comment (sfwn) [inline internal/apps/dop/providers/efficiency_measure/efficiency_query.go:238]: format it please.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go (hunk 2 above)
Comment (jing-zhi) [reply internal/apps/dop/providers/efficiency_measure/efficiency_query.go:238]: ok

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go (hunk 1 above)
Comment (jing-zhi) [reply internal/apps/dop/providers/efficiency_measure/efficiency_query.go:46]: ok

**Review by sfwn — APPROVED** [review]

**Pull request merged**